tests/TestHelpers/WebDavHelper.php

```diff
@@ -179,8 +179,10 @@ public static function getBodyForPropfind(?array $properties): string {
 	 * @param string|null $type
 	 * @param int|null $davPathVersionToUse
 	 * @param string|null $doDavRequestAsUser
+	 * @param bool|null $nonExistentUser
```

Comment:
the default is `false` so I think, better to not allow null
```suggestion
	 * @param bool $nonExistentUser
```